### ISSUE TITLE
Post action cleanup when removing uncompressed packages?

### ISSUE THREAD

**Issue by mroeschke** [mamba-org/setup-micromamba#68]
cc @jonashaag @pavelzw thanks for adding the setup-micomamba action to pandas

Noticing this error when this action is being cleaned up:

https://github.com/pandas-dev/pandas/actions/runs/4961741352/jobs/8879042319

```
Post job cleanup.
Removing uncompressed packages to trim down cache folder...

node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, scandir 'C:\Users\runneradmin\micromamba\pkgs'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'scandir',
  path: 'C:\\Users\\runneradmin\\micromamba\\pkgs'
}
```

**Comment by pavelzw** [comment]
Thanks for letting me know! This happened because there was a cache hit for the environment and a miss for the downloads resulting in the action wanting to cleanup a nonexistent folder. I'll implement a fix tomorrow!

**Comment by pavelzw** [comment]
@mroeschke this is fixed in `mamba-org/setup-micromamba@v1.4.1`. Since you used `@v1`, you don't need to update anything. Thanks again for the bug report :)

**Comment by mroeschke** [comment]
Thanks for the quick fix!
